examples/jira-scan-workflow/versions/0.0.1/workflowTemplate.yaml

```diff
@@ -0,0 +1,520 @@
+                  value: '{{ inputs.parameters.CF_API_KEY_SECRET_KEY }}'
+                - name: IMAGE_NAME
+                  value: '{{ inputs.parameters.IMAGE_NAME }}:{{ inputs.parameters.TAG }}'
+                - name: IMAGE_SHA
+                  value: '{{ inputs.parameters.IMAGE_SHA }}'
+                - name: GIT_PROVIDER
+                  value: '{{ inputs.parameters.GIT_PROVIDER }}'
+                - name: REPO
+                  value: '{{ inputs.parameters.REPO_URL }}'
+                - name: BRANCH
+                  value: '{{ inputs.parameters.REVISION }}'
+                - name: GITHUB_TOKEN_SECRET_NAME
+                  value: '{{ inputs.parameters.GIT_TOKEN_SECRET }}'
+                - name: GITHUB_TOKEN_SECRET_KEY
+                  value: '{{ inputs.parameters.GITHUB_TOKEN_SECRET_KEY }}'
+
+          # enrich image with jira info
+          # https://codefresh.io/argohub/workflow-template/CSDP-metadata
+          - name: image-enricher-jira-info
+            templateRef:
+              name: argo-hub.codefresh-csdp.1.1.0
+              template: image-enricher-jira-info
+            arguments:
+              parameters:
+                - name: CF_API_KEY
+                  value: '{{ inputs.parameters.CF_API_KEY }}'
+                - name: CF_API_KEY_SECRET_KEY
+                  value: '{{ inputs.parameters.CF_API_KEY_SECRET_KEY }}'
+                - name: IMAGE_NAME
+                  value: '{{ inputs.parameters.IMAGE_NAME }}:{{ inputs.parameters.TAG }}'
+                - name: IMAGE_SHA
+                  value: '{{ inputs.parameters.IMAGE_SHA }}'
+                - name: JIRA_PROJECT_PREFIX
+                  value: '{{ inputs.parameters.ISSUE_PROJECT }}'
+                - name: JIRA_MESSAGE
+                  value: '{{ inputs.parameters.ISSUE_DESCRIPTION }}'
+                - name: JIRA_HOST_URL
+                  value: '{{ inputs.parameters.JIRA_BASE_URL }}'
+                - name: JIRA_API_TOKEN_SECRET
+                  value: '{{ inputs.parameters.JIRA_USERNAME }}'
+                - name: JIRA_API_TOKEN_SECRET_KEY
+                  value: '{{ inputs.parameters.JIRA_API_KEY_SECRET_KEY }}'
+                - name: JIRA_EMAIL_SECRET_KEY
+                  value: '{{ inputs.parameters.JIRA_USERNAME_SECRET_KEY }}'
+
+          # test the image
+          - name: test-image
+            template: test
+            arguments:
+              parameters:
+                - name: IMAGE
+                  value: '{{ inputs.parameters.IMAGE_NAME }}:{{ inputs.parameters.TAG }}'
+
+          # scan the repo
+          # https://codefresh.io/argohub/workflow-template/sonar
+          - name: scan-repo
+            dependencies: [test-image]
+            templateRef:
+              name: argo-hub.sonar.0.0.1
+              template: sonar-scanner
+            arguments:
+              artifacts:
+                - name: REPO
+                  from: "{{tasks.clone-step-repo.outputs.artifacts.REPO}}"
```

Comment:
this should be "tasks.clone-step-repo.outputs.artifacts.repo" in lower case as defined in the block

`      outputs:
        artifacts:
          - name: repo
            path: /tmp/repo
`
in the clone template
How were you able to make it run?